samples/create_basic_vm.py

```diff
@@ -0,0 +1,100 @@
+# Description: This sample creates basic virtual machine
+# taking host, datastore and folder as parameters
+#
+# Prerequisite: Folder of type 'VirtualMachine' has to be created
+# if not already present
+
+import atexit
+
+from pyVim import connect
+from pyVmomi import vmodl
+from pyVmomi import vim
+
+from tools import cli
+from tools.pchelper import get_container_view, get_obj
+from tools import vm
+from tools.tasks import wait_for_tasks
+
+
+def get_args():
+    parser = cli.build_arg_parser()
+
+    parser.add_argument('-vn', '--vm-name', required=True,
+                        help="Name of the VirtualMachine to create.")
+    parser.add_argument('-hn', '--host-name', required=True,
+                        help="Name of the host where vm will reside")
+    parser.add_argument('-dn', '--datastore-name', required=True,
```

Comment:
is it possible to make datastore name optional. If user doesnt pass datastore name, can we retrieve it from hostname provided?